Tests/Microsoft.AppCenter.Test.UWP/AppCenterTest.cs

```diff
@@ -108,40 +108,6 @@ public void TestCorrectDatabasePathPassedFromStorageInitialization()
             }
         }
 
-        [TestMethod]
-        public void TestDbInitializationFolderNotExists()
-        {
-            // Make sure database directory does not exist before test.
-            if (Directory.Exists(Constants.AppCenterDatabasePath))
-            {
-                new DirectoryInfo(Constants.AppCenterDatabasePath).Delete(false);
-            }
-            using (var storageMock = new Storage.Storage())
-            {
-                storageMock.WaitOperationsAsync(TimeSpan.FromSeconds(10)).Wait();
-
-                // Verify that database is created inside local app data folder, and not locally.
-                Assert.IsTrue(File.Exists(Path.Combine(Constants.LocalAppData, Constants.AppCenterDatabaseFilename)));
-            }
-        }
-
-        [TestMethod]
-        public void TestDbInitializationFolderExists()
-        {
-            // Make sure database directory exists before test.
-            if (!Directory.Exists(Constants.AppCenterDatabasePath))
-            {
-                new DirectoryInfo(Constants.AppCenterDatabasePath).Create();
```

Comment:
`AppCenterDatabasePath` is a path to db file, not directory. DB file is placed in root `LocalAppData` without permissions to delete or create it. Deleting invalid test case